gateways/firstdata/request_builders.go

```diff
@@ -0,0 +1,62 @@
+package firstdata
+
+import (
+	"strconv"
+
+	"github.com/BoltApp/sleet"
+)
+
+func buildAuthRequest(authRequest *sleet.AuthorizationRequest) (*Request, error) {
```

Comment:
Also, not sure if this is because I mistakenly did this in other areas but if there is no error lets just return back *Request